tests/testpm.c

```diff
@@ -38,15 +38,13 @@ int main(int argc, char * argv[]) {
     //ADD NCDM
     double m_ncdm[3] = {0.05, 0.02, 0.01};
     int n_ncdm = 3;
-    FastPMncdmInitData* nid = fastpm_ncdm_init_create(config->boxsize, m_ncdm, n_ncdm, 0.6774, 9., 10, 2, 0);
+    FastPMncdmInitData* nid = fastpm_ncdm_init_create(config->boxsize, m_ncdm, n_ncdm, 0.6774, 9., 1, 1, 0);
     
-    int f_subsample_1d = 4;
-    int f_subsample_3d = f_subsample_1d*f_subsample_1d*f_subsample_1d;
```

Comment:
this subsample factor doesn't seem to be used; yet makes a huge total_np_ncdm that slows down the test (due to throttling in decompose).

Reply:
Hmmm, not sure why it seems not to be used, but no harm in removing it for the test.